### ISSUE TITLE
Fuzzing search doesn't work properly

### ISSUE THREAD

**Issue by huanggze** [kubesphere/console#740]
I can find out `PrometheusRules` by type `prom`, but no luck with `Prom`.

![image](https://user-images.githubusercontent.com/22350668/87291205-901b5080-c531-11ea-96ae-f7832709af11.png)
![image](https://user-images.githubusercontent.com/22350668/87291244-9b6e7c00-c531-11ea-8328-ab2600b0a288.png)

**Comment by zryfish** [comment]
yeah, keyword currently only searches in the name field, Prometheus is Kind, let's see what we can do.
/assign
